### PR TITLE
Documentation for a new integration for air-Q devices (air quality sensors)

### PR DESCRIPTION
**Pull request by Sibgatulin**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/76999
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3597
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Please rebase your branch to the `next` branch, because there are unrelated commits from the `current` branch in your PR.

**Comment by Sibgatulin** [comment]
Thanks for the quick review @klaasnicolaas! Sorry, my bad. 
Since this PR effectively contributes a single commit, I cherry-picked it on top of `next`, and force-pushed. I hope it does the trick.

**Review by ChristianHeimke — DISMISSED** [review]

File: source/_integrations/airq.markdown [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: "air-Q"
+description: "Instructions on how to integrate Air-Q into Home Assistant"
+ha_category:
+  - Environment
+  - Health
+  - Sensor
+ha_release: '2022.09'
```
Comment (frenck) [inline source/_integrations/airq.markdown]:
```suggestion
ha_release: '2022.12'
```

File: source/_integrations/airq.markdown [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  - "@dl2080"
+  - "@Sibgatulin"
+ha_domain: airq
+ha_config_flow: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+This platform allows to integrate the sensors, provided by your [Air-Q](https://www.air-q.com/) device, into Home Assistant.
```
Comment (frenck) [inline source/_integrations/airq.markdown]:
```suggestion
This integration allows integrating the sensors, provided by your [Air-Q](https://www.air-q.com/) device, into Home Assistant.
```

File: source/_integrations/airq.markdown [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+{% include integrations/config_flow.md %}
+
+During the configuration the user is prompted for the IP address of the device or the first 5 characters of the serial number, as well as the device password.
+
+For this integration to communicate with the device, both must be connected to the same WiFi network.
```
Comment (frenck) [inline source/_integrations/airq.markdown]:
```suggestion
During the configuration, the user is prompted for the IP address of the device or the first 5 characters of the serial number, as well as the device password.

For this integration to communicate with the device, both must be connected to the same Wi-Fi network.
```

File: source/_integrations/airq.markdown [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+| Pressure             | hPa                 |
+| SO2                  | µg/m³               |
+| Temperature          | °C                  |
+| VOC[^voc]            | ppb                 |
+
+[^rel]: Relative
+
+[^pm]: Correspond to concentrations of particulates with diameter less than 1µm, 2.5µm, and 10µm respectively
+
+[^voc]: Volatile organic compounds
```
Comment (frenck) [inline source/_integrations/airq.markdown]: These don't work for our website/documentation.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by Sibgatulin** [comment]
Thanks for the review, @frenck. I fixed the issues (+ extended the sensor list, as it grew during the parent PR development)

**Review by frenck — APPROVED** [review]
Thanks, @Sibgatulin 👍

**Comment by Sibgatulin** [comment]
Thanks for the prompt merge! :rocket:

**Pull request merged**